src/utils.ts

```diff
@@ -0,0 +1,23 @@
+const cardanoCrypto = require('cardano-crypto.js');
```

Comment:
Why `src/utils.ts` when there is a `src/wallet/tron.crypto.ts`? Why not `src/wallet/cardano.crypto.ts`?